pages/news/nightstand.md

```diff
@@ -0,0 +1,39 @@
+
+<p>A tired traveler arrives at a hotel late at night and finds there is no clock in the room.  Fortunately, this person has an AsteroidOS watch with nightstand mode.  She uses the settings to select the kind of display that she would like to use for this and enables the mode.  Then she puts the watch on its charging stand where it acts as sort of a bedside clock, prominently featuring the state of charge and the current time.  She wears glasses and does not want to have to put them on to see the time, so she chooses a watchface that is high contrast and features large numbers for the time.  Even with that, it's not always that easy to see the time, so it is configured so that at 3AM with no glasses, she can pick up the watch and bring it closer to note the time (ugh!  3AM!) and replace it on the stand within some configurable period of time with the watch not leaving nightstand mode.
+
+<h2>Requirements</h2>
+<p><em>Note:</em>Numbering is arbitrary and not meant to signify priority; just to provide a convenient handle for conversation.</p>
+<ol>
+<li>When nightstand mode is enabled, and the charger is connected (i.e. nightstand mode = on) change the display to show state of charge</li>
+<li>When nightstand mode is on, display the time in a way that's clear even without glasses</li>
+<li>The user should be able to enable/disable nightstand mode</li>
+<li>The user should be able to select/configure the nightstand display</li>
+<li>The user should be able to disconnect the charger for a brief period without changing from the nightstand display</li>
+<li>After the brief period expires, the display should revert to the regular watchface</li>
+<li>The duration of the brief period should be configurable from 0 to, say, 30 seconds</li>
+<li>When nightstand mode is on, the display should rotate based on orientation</li>
+</ol>
+
+<h2>Implementation</h2>
+<p>All of the requirements except rotation based on orientation are now implemented.</p>
+<p>After some discussion among developers, the way it works now is that if Nightstand mode is active and the watch is placed on a charger, by default, the currently selected watchface will be used for this mode.  Typically, this means that each watchface has sort of an alternative mode that adds some kind of charge indicator ring.</p>
+<p>To accomodate our glasses-wearing user as described in the scenario above, one can also select a custom watchface that will be used just when Nightstand mode is active.  This allows users to choose a complex and information-filled watchface during the day, but an easier-to-read version for nightstand mode.</p>
+<p>This mode is most useful when paired with Always-on display.</p>
+
+<h2>Your ideas are welcome!</h2>
+<p>Many of the features that are now implemented in AsteroidOS come from suggestions from users.  If you have ideas for a new feature or comments or questions on this one, drop by <a href="https://matrix.to/#/#Asteroid:matrix.org">Matrix chat</a> and say hello.</p>
```

Comment:
Would "drop by Matrix" or "drop by the Matrix chat" work better than "drop by Matrix chat" ? idk you're the native speaker :)

Reply:
Any of the three works grammatically in English.  I didn't use the first because people not already familiar with Matrix might not know what it is.